arrow-core/src/main/kotlin/arrow/core/extensions/option/apply/OptionApply.kt

```diff
@@ -100,8 +93,8 @@ fun <A, B, Z> Kind<ForOption, A>.map2Eval(
 @Deprecated(
   "@extension kinded projected functions are deprecated",
   ReplaceWith(
-  "map(arg0, arg1, arg2)",
-  "arrow.core.Option.map"
+  "Option.mapN(arg0, arg1) { a, b -> arg2(Tuple2(a, b)) }",
+    "arrow.core.Option", "arrow.core.Tuple2", "arrow.core.mapN"
```

Comment:
I think mapN import is not needed
```suggestion
    "arrow.core.Option", "arrow.core.Tuple2"
```